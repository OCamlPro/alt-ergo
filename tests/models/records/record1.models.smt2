(set-logic ALL)
(set-option :produce-models true)
(declare-datatype Pair
((pair (first Int) (second Int))))
(declare-const a Pair)
(assert (= (first a) 5))
(check-sat)
(get-model)
