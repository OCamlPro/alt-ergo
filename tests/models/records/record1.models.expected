
unknown
(
  (define-fun a () Pair (pair 5 0))
)
